Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -993,21 +995,26 @@ def get_remote_data_command(service, args, close_incident):
 
     for item in results.ResultsReader(service.jobs.oneshot(search)):
         updated_notable = parse_notable(item, to_dict=True)
+    delta = {field: updated_notable.get(field) for field in INCOMING_MIRRORED_FIELDS if updated_notable.get(field)}
+
+    if delta:
+        demisto.debug('notable {} delta: {}'.format(notable_id, delta))
+        if delta.get('status') == '5' and close_incident:
+            demisto.info('Closing incident related to notable {}'.format(notable_id))
+            entries.append({
+                'Type': EntryType.NOTE,
+                'Contents': {
+                    'dbotIncidentClose': True,
+                    'closeReason': 'Notable event was closed on Splunk.'
+                },
+                'ContentsFormat': EntryFormat.JSON
+            })
+
+        demisto.debug('Updated notable {}'.format(notable_id))
+    else:
+        demisto.debug('no delta was found for notable {}'.format(notable_id))
```

Comment:
Is this intentionally outside the for loop?

Reply:
yes, the loop is just to get the most update version of the notable

Reply:
So why do you need `entries` to be a list?